Scripts/create-release-tag.sh

```
@@ -0,0 +1,234 @@
+    # Update MARKETING_VERSION using sed
+    if sed -i '' "s/MARKETING_VERSION = [0-9]*\.[0-9]*\.[0-9]*;/MARKETING_VERSION = $new_version;/g" "$project_file"; then
+        print_success "Updated MARKETING_VERSION to $new_version ✓"
+    else
+        print_error "Failed to update MARKETING_VERSION"
+        # Restore backup
+        mv "${project_file}.backup" "$project_file"
+        exit 1
+    fi
+
+    # Verify the update
+    local updated_count=$(grep -c "MARKETING_VERSION = $new_version;" "$project_file" || echo "0")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)